arm/Microsoft.Synapse/workspaces/.bicep/nested_cmk.bicep

```diff
@@ -0,0 +1,37 @@
+param encryptionKeyVaultName string
+param encryptionKeyVaultResourceGroupName string
+param encryptionKeyName string
+param workspaceName string
+param workspacePrincipalId string
+
+var keyVaultUrl = 'https://${encryptionKeyVaultName}.vault.azure.net/keys/${encryptionKeyName}'
```

Comment:
> Environment URLs should not be hardcoded. Use the environment() function to ensure compatibility across clouds. Found this disallowed host: "vault.azure.net"bicep corehttps://aka.ms/bicep/linter/no-hardcoded-env-urls

The important part of this Pester warning is to leverage `environments()` instead of the plain `vault.azure.net` to enable compatibility with other clouds